### PR TITLE
Refactor/line item68 CohortComparisonApp and MutationFrequencyApp

### PR DESCRIPTION
**Pull request by ocshawn**
## Description
Fixing issues in jira: [PEAR-1694](https://gdc-ctds.atlassian.net/browse/PEAR-1694) item 9 and 10
Number of 508 fixes to Repository Page:
- [9 a fix keybord nav on radio buttons cohort comp page](https://github.com/NCI-GDC/gdc-frontend-framework/commit/20c2280d2ff99306d255516489b3926bb1a0cca7)
- [10 a ii fix label ID typo](https://github.com/NCI-GDC/gdc-frontend-framework/commit/404e7fb5da8501277cadbbee2241f5edf2d036d1)
- [10 b add label to cancer genes toggle](https://github.com/NCI-GDC/gdc-frontend-framework/commit/33f60a4152b9e23566a7665f62943090117010e7)
- [10 c fix download btn label](https://github.com/NCI-GDC/gdc-frontend-framework/commit/a84f25f44c5c1452caacd09aedc3131d9fe14d6d)
- [10 e i changed cohort column selectors to use mantine switch component](https://github.com/NCI-GDC/gdc-frontend-framework/commit/9b664db7bef0da0f1dc62e6fcf5272f34a79d981)
- [10 e ii 1 remove Summery Description more btn icon label](https://github.com/NCI-GDC/gdc-frontend-framework/commit/135a1b478f80a4e79bb13c3f6476af712e68dc8a)

## Checklist

- [x] Added proper unit tests
- [x] Left proper TODO messages for any remaining tasks
- [x] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

### CODE REVIEW

File: packages/portal-proto/src/features/facets/ToggleFacet.unit.test.tsx [hunk 1]
```diff
@@ -30,9 +30,7 @@ describe("<ToggleFacet />", () => {
       />,
     );
 
-    const ctrl = getByRole("checkbox", {
-      name: "toggle facet value",
-    });
+    const ctrl = getByTestId("toggle-facet-value");
```
Comment (amlehman) [inline packages/portal-proto/src/features/facets/ToggleFacet.unit.test.tsx:33]: Was this selector not working anymore? General convention for react testing library is to prefer selecting by role over test id: https://testing-library.com/docs/queries/about#priority

File: packages/portal-proto/src/features/GenomicTables/GenesTable/TableComponents/GenesTableCohort.tsx [hunk 2]
```diff
@@ -48,8 +48,6 @@ const GenesTableCohort = ({
       tooltip={
         isDemoMode
           ? "Feature not available in demo mode"
-          : toggledGenes.includes(geneID)
```
Comment (amlehman) [inline packages/portal-proto/src/features/GenomicTables/GenesTable/TableComponents/GenesTableCohort.tsx]: Why was this removed?

File: packages/portal-proto/src/features/facets/ToggleFacet.unit.test.tsx (hunk 1 above)
Comment (ocshawn) [reply packages/portal-proto/src/features/facets/ToggleFacet.unit.test.tsx:33]: it was not working because the aria label was remove in favor of a label with the "for" which i am updating to the more correct htmlFor

File: packages/portal-proto/src/features/GenomicTables/GenesTable/TableComponents/GenesTableCohort.tsx (hunk 2 above)
Comment (ocshawn) [reply packages/portal-proto/src/features/GenomicTables/GenesTable/TableComponents/GenesTableCohort.tsx]: good catch, placing it back. I know i was removing similar from aria labels, don't know why this was changed as its a tooltip

**Comment by cemile-ctds** [comment]
The toggle button is associated with more than one label. Please refer to the previously created ticket that is related to this issue: https://gdc-ctds.atlassian.net/browse/PEAR-1764

The remaining reported issues that have been fixed look good to me.

**Comment by ocshawn** [comment]
@cemile-ctds this looks to be a separate issue, this code only has one toggle button under it and is not the same code as that page

**Comment by cemile-ctds** [comment]
> @cemile-ctds this looks to be a separate issue, this code only has one toggle button under it and is not the same code as that page

I'm not sure if they have the same code or share the same component, but the issue is identical. Please refer to the screenshot below for reference.
Screenshot 2024-01-31 at 12 10 27

**Review by amlehman — APPROVED** [review]

**Comment by ocshawn** [comment]
@cemile-ctds reverted Swich code will work on updating that component in a separate PR
